sdk/eventgrid/eventgrid/README.md

```diff
@@ -255,42 +255,31 @@ await client.send([
 `EventGridDeserializer` can be used to deserialize events delivered by Event Grid. When deserializing an event, you need to know the schema used to deliver the event. In this example we have events being delivered to an Azure Service Bus Queue in the Cloud Events schema. Using the Service Bus SDK we can receive these events from the Service Bus Queue and then deserialize them using `EventGridDeserializer` and use `isSystemEvent` to detect what type of events they are.
```

Comment:
since we're not showing an end-to-end but rather just the deserialization, this paragraph should be cleaned up so that it makes sense

Reply:
@xirzec Thanks. I have updated the paragraph.